src/styles/components/code/styles.less

```diff
@@ -6,6 +6,7 @@
       border: @code-block-prettyprint-border-width @code-block-prettyprint-border-style @code-block-prettyprint-border-color;
       border-radius: @code-border-radius;
       border-style: @code-block-prettyprint-border-style;
+      height: @code-block-prettyprint-height;
```

Comment:
Instead of setting `height: 1px`, a cleaner way to fix this would be `flex-basis: 0;`